src/app-layout/visual-refresh-toolbar/navigation/styles.scss

```diff
@@ -30,17 +27,16 @@
     display: none;
   }
 
-  & > .content-container {
+  & > .navigation {
     flex-grow: 1;
-  }
-
-  /*
-  A non-semantic node is added with a fixed width equal to the final Navigation
-  width. This will create the visual appearance of horizontal movement and
-  prevent unwanted text wrapping.
-  */
-  > .animated-content {
+    block-size: 100%;
+    overflow-y: auto;
+    /* stylelint-disable-next-line plugin/no-unsupported-browser-features */
+    overscroll-behavior-y: contain;
```

Comment:
I see that it prevents overscroll when using a trackpad (and I assume also when on a touchscreen). The video capture below illustrates this overscroll behavior, which can only be reproduced by removing this line. Note that I am just moving it around so that it keeps belonging to the actual scrollable element, and I see that it has the same effect indeed. This was introduced more than two years ago and is used in navigations and tools for Visual Refresh, with and without toolbar. I would not change this behavior in the scope of this PR.

https://github.com/user-attachments/assets/a660b1e7-d248-44ef-b3e2-0920286d84c0

Reply:
Ah I had missed that it was already present in the other class. Thanks!